app/ai_backend/test_genesis_connector.py

```
@@ -1300,47 +2247,41 @@
 
     def test_payload_formatting_performance(self):
         """
-        Measures the time taken to format payloads of increasing sizes and asserts that formatting completes within one second for each size.
+        Test that formatting payloads of various sizes completes in under one second and produces a non-None result.
         """
         import time
         
-        sizes = [10, 100, 1000, 5000]
+        timezone_test_cases = [
+            datetime.now(timezone.utc),  # UTC
```

Ruff (0.11.9)

Undefined name `timezone`

(F821)